docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,76 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+If `true`, the default input is in read-only mode and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput
+    source="tags_ids"
+    reference="tags"
+    readOnly
+/>
+```
+
+{% endraw %}
+
+**Warning**: The `readOnly` prop refer to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you defined `children` you have to specify the `readOnly` prop on each input:
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput source="tags_ids" reference="tags">
+    <SelectArrayInput readOnly />
+</ReferenceArrayInput>
+```
+
+{% endraw %}
+
+## `disabled`
+
+If `true`, the default input is disabled and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput
+    source="tags_ids"
+    reference="tags"
+    disabled
+/>
+```
+
+{% endraw %}
+
+**Tip**: The form framework used by react-admin, react-hook-form, [considers](https://github.com/react-hook-form/react-hook-form/pull/10805) that a `disabled` input shouldn't submit any value. So react-hook-form sets the value of all `disabled` inputs to `undefined`. As a consequence, a form with a `disabled` input is always considered `dirty` (i.e. react-hook-form considers that the form values and the initial record values are different), and it triggers [the `warnWhenUnsavedChanges` feature](./EditTutorial.md#warning-about-unsaved-changes) when leaving the form, even though the user changed nothing. The workaround is to set the `disabled` prop on the underlying input component, as follows:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** I'm not sure about adding this paragraph. It seems like we are blaming RHF, but they are following the [readonly specs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled). What do you think @slax57